certified-connectors/Tendocs Documents/apiDefinition.swagger.json

```diff
@@ -108,9 +108,243 @@
             "description": "Something has gone wrong, please contact support@tendocs.com"
           }
         },
-        "summary": "Convert Document To New File Format",
+        "summary": "Convert Document To New File Format (PDF/HTML/TXT)",
```

Comment:
Summary should contain restricted characters, kindly update. 

Summary string contains any of the restricted characters [/\<>*%&?]